src/app/utils/PickItemModel.js

```diff
@@ -16,22 +21,22 @@ function PickItemModel() {
   };
 
   this.pickItem = data => {
-    let item = {};
+    const item = {};
 
     if (!data) {
       return item;
     }
 
     item.id = data.id;
     item.type = data.type;
-    
+
     item.title = data.attributes.title;
```

Comment:
I would wrap all these in a if (data.attributes) conditional check because you will get a `cannot read catalog-slug property of undefined` if `data.attributes doesn't exist. I don't see that check...

Reply:
These files will go once we get a new data model so I rather not spend time updating these.